bitnami/redis-cluster/templates/svc-cluster-external-access.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.cluster.externalAccess.enabled }}
+{{- $fullName := include "redis-cluster.fullname" . }}
+{{- $nodesCount := .Values.cluster.nodes | int }}
+{{- $root := . }}
+
+{{- range $i, $e := until $nodesCount }}
+{{- $targetPod := printf "%s-%d" (printf "%s" $fullName) $i }}
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "redis-cluster.fullname" $ }}-{{ $i }}-svc
+  labels: {{- include "redis-cluster.labels" $ | nindent 4 }}
+    pod: {{ $targetPod }}
+  annotations: {{ include "common.tplValues.render" ( dict "value" $root.Values.cluster.externalAccess.service.annotations "context" $) | nindent 4 }}
```

Comment:
Be careful, in the common chart there's `common.tplvalues.render` not `common.tplValues.render`. Make sure you change it.